.buildkite/scripts/packaging/package.sh

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+source .buildkite/scripts/packaging/package-util.sh
+
+IMG_POSTFIX="-SNAPSHOT"
+VARIANTS=("" "-ubi" "-oss")
+VERSION="$(make get-version)"
+SOURCE_TAG+="${VERSION}${IMG_POSTFIX}"
+TARGET="observability-ci/${BEATS_PROJECT_NAME}"
+
+echo "--- Creating package"
+mage -d "${BEATS_PROJECT_NAME}" package
+
+echo "--- Distribution list"
+dir="${BEATS_PROJECT_NAME}/build/distributions"
+buildkite-agent artifact upload "$dir/*.tar.gz;$dir/*.tar.gz.sha512"
+
+echo "--- Docker image list"
+docker images
+
+define_tags
+check_is_arm
```

Comment:
I think the name of the function does not reflect what it does.

Reply:
I think being explicit rather than using a function can help to understand what it does.
```suggestion

targetSuffix=""
if [[ ${HW_TYPE} == "aarch64" || ${HW_TYPE} == "arm64" ]]; then
  targetSuffix="-arm64"
fi

```